src/modules/common/data/BitField.ts

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+
+    The class that defines and parses the BitField of a block.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+import { Base } from './Base';
+
+/**
+ * The class that defines and parses the BitField of a block.
+ * Convert JSON object to TypeScript's instance.
+ * An exception occurs if the required property are not present.
+ */
+export class BitField extends Base
```

Comment:
Isn't there a package for this ? There must be.

Reply:
I looked it up and used it, and there was an error with it.

Reply:
What package ? What error ?

Reply:
https://www.npmjs.com/package/ts-transformer-keys
This is the best fit, but it doesn't work.

Reply:
Extracting the members of the class in an array is an issue in TypeScript.